Sources/Engine/Rendering/SelectOnRender.cpp

```diff
@@ -165,8 +165,7 @@ void SelectVertexOnRender(CBrushVertex &bvx, const PIX2D &vpix)
   // if selecting with lasso
   } else {
     // if the vertex is set in lasso buffer
-    if (_pubLassoBuffer!=NULL
-      &&_pubLassoBuffer[vpix(2)*_pixSizeI+vpix(1)]) {
```

Comment:
Why did you remove the NULL test?

Reply:
Because the if from line [151](https://github.com/ptitSeb/Serious-Engine/blob/19f42367ccbb02763328d13274cfd6b92188f043/Sources/Engine/Rendering/SelectOnRender.cpp#L151) already covers the _pubLassoBuffer==NULL and the else on line 166 means that _pubLassoBuffer will always be !=NULL hence, the condition is redundant

Reply:
👍